src/components/Hero.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Typography, Grid, useMediaQuery } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import HeroImage from '../../public/hero-img.png';
+import theme from '../theme';
+import { LandingPageResponse } from '../utils/types';
+
+import GradientBorderDivider from './GradientBorderDivider';
+
+interface HeroProps {
+  data: LandingPageResponse;
+}
+
+const Hero: React.FC<HeroProps> = ({ data }) => {
+  const { heroSection } = data;
+  const imageAlt = heroSection.images[0].alt;
+
+  const isMobile = useMediaQuery(theme.breakpoints.down(750));
+
+  return (
+    <>
+      <Grid
+        container
+        spacing={0}
+        justifyContent="center"
+        alignItems="center"
+        sx={{
+          padding: '21px 16px 48px 16px',
+          '@media (min-width: 600px)': { padding: '75px 16px' },
+          maxWidth: isMobile ? '100%' : '1100px',
+          margin: '0 auto',
+        }}
+        direction={isMobile ? 'column' : 'row'}
+      >
+        <Grid item xs={12} sm={5} style={{ padding: 0, margin: 0 }}>
```

Comment:
why Grid here? Just curious why you use Grid inside Grid, and not just a Box?

Reply:
Hi @stepsen89 
I referenced from the MUI docs https://mui.com/material-ui/react-grid/
It seemed like a common pattern.

![image](https://github.com/Women-Coding-Community/wcc-frontend/assets/29165867/9385065e-934c-4930-8392-52759864a133)

Reply:
that's fine - as we go along with MUI we get more experienced - was just wondering as I looked up the same with Box to see if we're able to use Box inside Box and that's also possible/common 😅